### PR TITLE
Feature/cu kd4rcp fang balance

### PR DESCRIPTION
**Pull request by telmiuse**
Shoot Trail Prefab
Atack Speed var to balance 
All applied on scene 1

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-kd4rcp Fang Shoot Balance](https://app.clickup.com/t/kd4rcp)

**Comment by isaacfdez** [comment]
I think you should merge Cristian's Branch when it is in develop before merging this one, as it has a lot of changes to the Player Controller Script that may overlap with yours

**Comment by Yonimevicio** [comment]
> 
> 
> I think you should merge Cristian's Branch when it is in develop before merging this one, as it has a lot of changes to the Player Controller Script that may overlap with yours

We talked about and we decided to merge it in my branch. I will merge it with my branch when it be in develop.

File: Penteract/Assets/Scripts/TrailScript.cpp
Comment (Jordi-Gil) [inline Penteract/Assets/Scripts/TrailScript.cpp:7]: Quita la , del final

File: Penteract/Assets/Scripts/PlayerController.h
Comment (Jordi-Gil) [inline Penteract/Assets/Scripts/PlayerController.h:122]: Ponlo en minúscula

**Review by Jordi-Gil — DISMISSED** [review]

**Review by Yonimevicio — DISMISSED** [review]

**Review by Jordi-Gil — APPROVED** [review]

**Review by Yonimevicio — DISMISSED** [review]

**Review by jcarlos0305 — APPROVED** [review]

**Review by Yonimevicio — APPROVED** [review]

**Review by Jordi-Gil — APPROVED** [review]

**Pull request merged**
